scripts/roles.json

```diff
@@ -796,6 +952,43 @@
     "requiredProps": [],
     "superClass": ["widget"]
   },
+  "comment": {
+    "abstract": false,
+    "accessibleNameRequired": false,
+    "childrenPresentational": false,
+    "nameFrom": ["author", "contents"],
+    "prohibitedProps": [],
+    "props": [
+      "aria-atomic",
+      "aria-braillelabel",
+      "aria-brailleroledescription",
+      "aria-busy",
+      "aria-controls",
+      "aria-current",
+      "aria-describedby",
+      "aria-description",
+      "aria-details",
+      "aria-dropeffect",
+      "aria-flowto",
+      "aria-grabbed",
+      "aria-hidden",
+      "aria-keyshortcuts",
+      "aria-label",
+      "aria-labelledby",
+      "aria-level",
+      "aria-live",
+      "aria-owns",
+      "aria-posinset",
+      "aria-relevant",
+      "aria-roledescription",
+      "aria-setsize"
+    ],
+    "relatedConcepts": [],
+    "requiredContextRole": [],
+    "requiredOwnedElements": [],
+    "requiredProps": [],
+    "superClass": ["article"]
```

Comment:
I disagree with the spec here. Subclassing on `article` is ontologically incorrect. But it is not ours to wonder why :)